example/test.py

```diff
@@ -61,13 +61,40 @@ def row_matcher(values):
     return MatchesSetwise(*map(row_matcher, expected_values))
 
 
+def find_prefork_worker_split(ps_rows):
+    """
+    Given a list of PsRow objects for a group of processes using a pre-fork
+    worker process model, finds the row for the "master" process as well as
+    a list of "worker" processes.
+    """
+    ppid_to_rows = {}
+    for row in ps_rows:
+        ppid_to_rows.setdefault(row.ppid, []).append(row)
+
+    # There should only be to ppids: the parent of the master and the master
```

Comment:
Sadly, it wouldn't be sufficiently accurate to write "Always two ppids there are; no more, no less. A master and an apprentice."